tests/test_circuit.py

```diff
@@ -857,6 +857,104 @@ def test_circuit_quoperator(backend):
     qo = c.quoperator()
     np.testing.assert_allclose(qo.eval_matrix(), c.matrix(), atol=1e-5)
 
+@pytest.mark.parametrize("backend", [lf("npb"), lf("tfb"), lf("jaxb")])
+def test_qir2cirq(backend):
+    try:
+        import cirq
+        from tensorcircuit.translation import perm_matrix
```

Comment:
we dont need ``perm_matrix`` in cirq test since cirq has the same result order as tc?

Reply:
Sure, forgot to delete that